packages/UI/.storybook/main.ts

```diff
@@ -33,9 +33,9 @@ const config: StorybookConfig = {
         esModuleInterop: false,
       },
       shouldExtractLiteralValuesFromEnum: true,
-      propFilter: () => true,
+      propFilter: (prop) => (prop.parent ? !/node_modules/.test(prop.parent.fileName) : true),
```

Comment:
This will filter out props that are coming `node_modules` from being in the docs.